assets/css/screen.css

```diff
@@ -851,7 +851,7 @@ make sure this only happens on large viewports / desktop-ish devices.
 .post-full-content blockquote,
 .post-full-comments,
 .footnotes {
-  min-width: 100%;
+  width: 100%;
```

Comment:
```suggestion
  min-width: 100%;
```

Reply:
I believe `min-width: 100%` is from the default Casper theme this theme is based on.

I tested with both `width: 100%` and `min-width: 100%` and there doesn't seem to be a difference, at least with the articles I have locally.

It may be fine to change this, but I think it'd be better to stick with the default here if it doesn't have an effect on the math equations.

What do you think? Also pinging @ahmadabdolsaheb for his input as well.

Reply:
I think width: 100% and min-width: 100% mean the same think at this point.

Reply:
Yes, you are both right. No need to set `width` instead of `min-width` to 100%. We can change it back :)